### PR TITLE
feat(): change design for proposal title and description

### LINKED ISSUE #1845: Refactor create proposal inputs (Question and description)

### PR DESCRIPTION
**Pull request by midgerate**
Fixes #1845

The image upload stuff will coming up later. 

https://www.loom.com/share/19f86109a7624becb390e46a5411d28a

### CODE REVIEW

**Comment by samuveth** [comment]
It would be better to have the input radius to be consistent with other inputs on `step 2`. Also the `400px` with not work well on all screens, better to use a `vh`

File: src/views/SpaceCreate.vue [hunk 1]
```diff
@@ -396,29 +416,68 @@ const needsPluginConfigs = computed(() =>
...
+                    {{ $t('create.uploadImageExplainer') }}
+                  </span>
+                </span>
+                <a
+                  class="relative inline"
+                  href="https://docs.github.com/github/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax"
+                  target="_blank"
+                  v-tippy="{ content: $t('create.markdown') }"
+                >
+                  <BaseIcon name="github" class="text-skin-text" />
```
Comment (samuveth) [inline src/views/SpaceCreate.vue:478]: Do you need me to add the markdown icon?

**Comment by samuveth** [comment]
The scrollbar is overlapping the textarea and border
image

You can solve this with overflow hidden on the container with the border, just like on the `BaseAutocomplete` component

**Comment by samuveth** [comment]
Border should be rounded at bottom just like at the top. If you need to add a footer slot to the Textarea component to achieve this.
image

**Comment by bonustrack** [comment]
The icon for github link should be the markdown icon, not the GitHub icon.

File: src/views/SpaceCreate.vue (hunk 1 above)
Comment (midgerate) [reply src/views/SpaceCreate.vue:478]: Yes.

**Review by samuveth — APPROVED** [review]
👍

**Pull request merged**